### PR TITLE
NEW: AD-1570 - enter referral code after secret phrase confirm

### PR DESCRIPTION
**Pull request by iryna-novykova-axon**

### CODE REVIEW

File: bladeui/js/pages/referralCode.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+          this.referralCodeField.value.length === 1 ? "SKIP" : "CONFIRM";
+        break;
+      default:
+        break;
+    }
+  }
+
+  handlePasteCode()
+  {
+    this.handleReferralCodeKeyPress();
```
Comment (maksym-dyachenko-axon) [inline bladeui/js/pages/referralCode.js]: you can use this function in event listener

File: run.sh [hunk 2]
```diff
@@ -8,5 +8,4 @@ docker cp blade:/app/blade.zip .
 
 docker cp blade:/app/devenv.chrome .
 
-docker stop blade && docker rm blade && docker rmi blade
-
+docker stop blade && docker rm blade
```
Comment (maksym-dyachenko-axon) [inline run.sh]: return docker rmi blade

File: bladeui/js/pages/referralCode.js (hunk 1 above)
Comment (iryna-novykova-axon) [reply bladeui/js/pages/referralCode.js]: done

File: run.sh (hunk 2 above)
Comment (iryna-novykova-axon) [reply run.sh]: done

**Comment by iryna-novykova-axon** [comment]
additional:
- hover effect on option (5, 10, 25)
- fixed button positions on referrals list and form view
- click out side drop down options closes it

File: bladeui/js/pages/referrals.js [hunk 3]
```diff
@@ -18,20 +18,27 @@ class Referrals extends BaseClass
     const optionsDropDown = document.getElementById("open-select-options");
     this.dropdown = document.getElementById("choose-option");
     this.emailListTarget = document.getElementById("referrals-list-content");
-    browser.storage.sync.get("bladeUserData", (data) =>
-    {
-      this.bearerToken = data.bladeUserData.token;
-      this.getReferralsInfo();
-    });
+    // in developer mode doesnt work requests
```
Comment (maksym-dyachenko-axon) [inline bladeui/js/pages/referrals.js]: remove this comment

File: bladeui/js/pages/referralCode.js [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+    this.mainActionButton.innerHTML = "SKIP";
+    this.referralCodeError.innerHTML = errorText;
+    this.referralCodeField.classList.add("input-invalid");
+  }
+
+  handleReferralCodeKeyDown(event)
+  {
+    const keyID = event.keyCode;
+    // check if entered referral code was removed completely
+    switch (keyID)
```
Comment (maksym-dyachenko-axon) [inline bladeui/js/pages/referralCode.js]: seems like if else be more obvious

File: bladeui/js/pages/referralsForm.js [hunk 5]
```diff
@@ -19,10 +19,10 @@ class ReferralsForm extends BaseClass
     this.emailError = document.getElementById("email-error");
     this.message = document.getElementById("message");
     this.form = document.getElementById("referral-form");
-    browser.storage.sync.get("bladeUserData", (data) =>
-    {
-      this.bearerToken = data.bladeUserData.token;
-    });
+    // browser.storage.sync.get("bladeUserData", (data) =>
```
Comment (maksym-dyachenko-axon) [inline bladeui/js/pages/referralsForm.js]: the same about comments here, please uncomment it

**Comment by iryna-novykova-axon** [comment]
close due to merge conflict, created new PR

**Pull request closed without merging**